library/gen_x12_837i.inc.php

```diff
@@ -0,0 +1,1244 @@
+<?php
+/**
+ * X12 837I
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2017 Jerry Padgett <sjpadgett@gmail.com>
+ * @license https://www.gnu.org/licenses/agpl-3.0.en.html GNU Affero General Public License 3
```

Comment:
is this code was derived from the other gen_x12, then should use the GPL3 license instead:
``````
* @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3